api/v1alpha4/cluster_types.go

```diff
@@ -127,10 +127,6 @@ type ClusterStatus struct {
 	// +optional
 	InfrastructureReady bool `json:"infrastructureReady"`
 
-	// ControlPlaneInitialized defines if the control plane has been initialized.
-	// +optional
-	ControlPlaneInitialized bool `json:"controlPlaneInitialized"`
-
```

Comment:
⚠️ ?

Reply:
Are we doing ⚠️ for anything & everything that is an API change from a3 to a4? If so, this PR is both 🐛 and ⚠️ - not sure what to do

Reply:
Worth to get the ⚠️ in place, so when we get to release notes we can write down the impact in addition to the bug fix